src/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <!-- Footer -->
+<footer class="page-footer font-small black-color" color="black">
+  <container class="text-left">
+      <row>
+        <col md="2">
+          <h5 class="text-uppercase mb-4 mt-3 font-weight-bold">AFRICAN INSTITUTE OF FINANCIAL MARKETS AND RISK MANAGEMENT</h5>
```

Comment:
what are these classes doing? Where are they being loaded from?

Reply:
I was doing some experimentation from the website where I did the tutorial for this from. I can tweak the syntax and see if no changes occur